analytics/pyproject.toml

```diff
@@ -9,6 +9,7 @@ version = "0.1.0"
 [tool.poetry.scripts]
 analytics = "analytics.cli:app"
 db-migrate = "analytics.cli:migrate_database"
+opportunity-load = "analytics.cli:ingest_opportunity_data"
```

Comment:
Not necessary, we can call the same function via app entrypoint. I don't remember how to do that offhand though

Reply:
I think what is already setup in the Makefile makes it work, the makefile isn't even running `poetry opportunity-load` - I think this line could just be deleted